tests/Stepper.test.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+
+import { render } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import { Stepper } from "components";
+
+const STEPS = [
+  { id: "1", label: "Connect your calendar" },
+  { id: "2", label: "Connect conferencing app" },
+  { id: "3", label: "Set availability" },
+];
+
+const getActiveClass = () =>
+  document.querySelector(".neeto-ui-stepper-item__wrapper--active");
```

Comment:
Instead of document queries, add a testId based on the index to the `div` and query using `screen.getByTestId`, then we can use `expect(screen.getByTestId('testid')).toHaveClass("")` for verifying the active state